### PR TITLE
Update Manubot and GitHub Actions methods

### LINKED ISSUE #621: Method TODO: Flesh out CI strategy in methods
**Is your feature request related to a problem? Please describe.**
@agitter is planning to update the methods, but @cgreene mentioned in #619 that it would especially great to mention the branch-based output strategy so that future projects know it's an option!

**Describe the solution you'd like**
Just making an explicit issue for what I believe is already on Tony's to do list: it would be great if he could look through the methods and provide a bit more insight 

**Describe alternatives you've considered**
I took a shot in #619 but it's not as detailed as would be ideal

**Additional context**

### PR DESCRIPTION
**Pull request by agitter**
### Description of the proposed additions or changes

Adds Methods section detail about Manubot, GitHub Actions, and the external resources.

### Related issues

Closes #621

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

I noticed one of the figures referenced in this section does not appear in the manuscript (#673).  I also deleted the comment about the RNA-seq methods because it looks like that part of the text and the corresponding figure were removed.  I tried archiving the repo on Software Heritage, but failed both times.  I would guess it's because of the huge files we have.  If we want to archive these scripts, we likely need to use a different service.

### CODE REVIEW

**Comment by agitter** [comment]
I started looking at the citation errors in the build log, but there are too many to address in this pull request.  We'll need to correct those before submission.

File: content/60.methods.md [hunk 1]
```diff
@@ -12,26 +12,36 @@ Below, we will describe the processes used to synthesize the literature.
...
-Similarly, Figure @fig:ebm-trials was generated based on data from the University of Oxford Evidence-Based Medicine (EBM) Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709].
-In both cases, frequency data was plotted using MatPlotLib in Python.
-Figure @fig:ebm-map was generated using the countries associated with the trials listed in the EBM's COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas. 
-Current version information for packages and software is available in https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml.
-
-<!-- Add in RNA-Seq analysis-->
+The combination of Manubot and GitHub Actions also made it possible to dynamically update information such as statistics and visualizations in the manuscript.
```
Comment (cgreene) [inline content/60.methods.md:26]: I know you say statistics twice, but it's so unusual that I think folks might read right by it. What do you think about including one of the statistics here and inputting the variable?

File: content/60.methods.md [hunk 2]
```diff
@@ -12,26 +12,36 @@ Below, we will describe the processes used to synthesize the literature.
...
+Figure @fig:ebm-map was generated using the countries associated with the trials listed in the COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas.
+<!-- Is this figure going to be included? #673 -->
+
+GitHub Actions runs a nightly workflow to update these external data and regenerate the statistics and figures for the manuscript.
+The workflow uses the GitHub API to detect and save the latest commit of the external data sources, which are both GitHub repositories.
+It then downloads versioned data from that snapshot of the external repositories and runs bash and Python scripts to calculate the desired statistics and produce the summary figures.
+The statistics are stored in JSON files that are accessed by Manubot to populate the values of placeholder template variables dynamically every time the manuscript is built.
+For instance, the template variable {% raw %}`{{ebm_trials_results}}`{% endraw %} in the manuscript is replaced by the actual number of clinical trials with results, {{ebm_trials_results}}.
```
Comment (cgreene) [inline content/60.methods.md:37]: Ahh - I see you did it here - very nice!

**Review by cgreene — APPROVED** [review]
Looks great! 🚀  I had one comment but you did what I suggested later on 😁

File: content/60.methods.md (hunk 2 above)
Comment (agitter) [reply content/60.methods.md:37]: Would it be better if I move the example earlier?

Comment (cgreene) [reply content/60.methods.md:37]: I don't have strong feelings - I made the first comment when I wanted to see it but then this came along with everything I was hoping to find.

Comment (agitter) [reply content/60.methods.md:37]: Sounds good.  I'm going to merge given your approval and can make new updates if @rando2 has any comments later.

**Pull request merged**